system/Filters/Filters.php

```diff
@@ -110,7 +111,40 @@ public function __construct($config, RequestInterface $request, ResponseInterfac
 		$this->config  = $config;
 		$this->request = &$request;
 		$this->setResponse($response);
+		if ((!is_array($this->config)) && ($this->config->discoverFilters))
```

Comment:
`$this->config` is a `\Config\Filters` instance, so, can use the following check:

```suggestion
              if (isset($this->config->discoverFilters) && $this->config->discoverFilters)
```

Reply:
Thanks you, have made changes this makes alot more sense